src/style.css

```diff
@@ -1,3 +1,82 @@
-body {
+* {
   margin: 0;
+  padding: 0;
+  box-sizing: border-box;
```

Comment:
The universal selector (*) is used here, which can impact performance. It's better to set styles only for elements that require them.